game-app/game-core/src/main/java/games/strategy/triplea/ui/menubar/help/MoveHelpMenu.java

```diff
@@ -20,53 +20,50 @@ Action buildMenu() {
           editorPane.setEditable(false);
           editorPane.setContentType("text/html");
           final String hints =
-              "<b> Selecting Units</b><br>"
-                  + "Left click on a unit stack to select 1 unit.<br>"
-                  + "ALT-Left click on a unit stack to select 10 units of that type in "
-                  + "the stack.<br>"
-                  + "CTRL-Left click on a unit stack to select all units of that type in the "
-                  + "stack.<br>"
-                  + "Shift-Left click on a unit to select all units in the territory.<br>"
-                  + "Left click on a territory but not on a unit to bring up a selection "
-                  + "window for inputting the desired selection.<br>"
-                  + "<br><b> Deselecting Units</b><br>"
-                  + "Right click somewhere not on a unit stack to unselect the last "
-                  + "selected unit.<br>"
-                  + "Right click on a unit stack to unselect one unit in the stack.<br>"
-                  + "ALT-Right click on a unit stack to unselect 10 units of that type in "
-                  + "the stack.<br>"
-                  + "CTRL-Right click on a unit stack to unselect all units of that type "
+              "<font size='+1'>Selecting Units</font><br>"
```

Comment:
+1 ?
Is this the same like a simple '1'?

Reply:
html font size is a number from 1 (small) to 7 (large). +1 means 1 size larger. 
It's deprecated in html5 which uses css.
On my win10 machine with Substance the default font size is '4'.

Reply:
Ah ok thanks for the clarification

Reply:
I should mention the display (JEditorPane) is in a JScrollPane, however the scrolling bars will not be displayed even if the text is larger than the screen as the viewport of JScrollPane is not specified. Not a problem here as the text will fit in screen. The JScrollPane could be removed.